<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="Line_Printf_Purchase">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>128</width>
     <height>96</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>96</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>120</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_7">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/MoralFU.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice8">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice5">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;2.5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice2">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;3.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_1">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg);
border-image: url(:/back/picture/COLA.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice7">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;5.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice3">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;2.5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_6">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/BingGna.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice4">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;6.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice6">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;3.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_4">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/back/picture/4c0495da4420c52972a8eaacdad86a6d.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/KingOldGood.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_3">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/BeautifulYearReach.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_9">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/HHHHHHH.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice1">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;2.5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="btn_item_5">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/RedTea.jpeg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_8">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/Plate.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="btn_item_2">
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65</width>
        <height>65</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back/picture/GreenTea.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="laPrice9">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;3.5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btn_item_7</zorder>
   <zorder>btn_item_5</zorder>
   <zorder>btn_item_9</zorder>
   <zorder>btn_item_3</zorder>
   <zorder>btn_item_2</zorder>
   <zorder>btn_item_4</zorder>
   <zorder>btn_item_6</zorder>
   <zorder>btn_item_1</zorder>
   <zorder>btn_item_8</zorder>
   <zorder>laPrice8</zorder>
   <zorder>laPrice5</zorder>
   <zorder>laPrice2</zorder>
   <zorder>laPrice7</zorder>
   <zorder>laPrice3</zorder>
   <zorder>laPrice6</zorder>
   <zorder>laPrice1</zorder>
   <zorder>laPrice9</zorder>
   <zorder>laPrice4</zorder>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>一键下单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Withdrawal">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>取消商品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_ShowWhatYouBuy">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>显示购物车</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
  <zorder>Line_Printf_Purchase</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
